var/spack/repos/builtin/packages/libffi/package.py

```diff
@@ -32,6 +32,11 @@ class Libffi(AutotoolsPackage):
     patch("clang-powerpc-3.2.1.patch", when="@3.2.1%clang platform=linux")
     # ref.: https://github.com/libffi/libffi/pull/561
     patch("powerpc-3.3.patch", when="@3.3")
+    patch(
+        "https://github.com/raw/libffi/libffi/pull/764.patch?full_index=1",
+        sha256="24a48b651679834682140aadfca59df4e9a7ed659c5da44b2daad4c87b966c6a",
+        when="@3.4.3:3.4.4",
+    )
```

Comment:
```suggestion
    patch(
        "https://github.com/libffi/libffi/commit/ce077e5565366171aa1b4438749b0922fce887a4.patch?full_index=1",
        sha256="070b1f3aa87f2b56f83aff38afc42157e1692bfaa580276ecdbad2048b818ed7",
        when="@3.4.3:3.4.4",
    )
```

I think the url has a redundant `raw/` bit? Since it's merged the commit can be referenced directly instead of the pull request, dunno if pull request urls stay forever.

Reply:
Thanks! updated